modules/test/src/main/java/com/devonfw/module/test/common/base/BaseTest.java

```diff
@@ -73,7 +73,7 @@ public final void tearDown() {
    */
   protected boolean isInitialSetup() {
 
-    return INITIALIZED;
+    return !initialized;
```

Comment:
Indeed this was inverse and has always been wrong. Seems I missed this in my review and it is a shame that nobody found all this earlier...